### DISCUSSION TITLE
How can I specify which monitor DOSBox Staging should open on under Linux and Sway?

### DISCUSSION THREAD

**Discussion by maxigaz** [dosbox-staging/dosbox-staging#2043] [Q&A] [answered]
I'm using Sway 1.7 under Arch Linux with two display outputs: VGA-1 and HDMI-A-1. DOSBox Staging 0.79.1 always opens on HDMI-A-1, regardless of which display is in focus. Is there a way to force it to open on VGA-1?

I've tried setting the environment variable `SDL_DISPLAYINDEX` with the values `0` and `1`, but it doesn't seem to make any difference.

**Comment by kcgen** [comment, marked as answer]
Try the `display` settings in your conf:

```ini
[sdl]
# display: Number of display to use; values depend on OS and user settings.
```
